### PR TITLE
Add lnl npu driver recommend version and enable cpu_lm_head on llama3

### PR DESCRIPTION
**Pull request by cyita**
## Description

Add lnl npu driver recommend version and enable cpu_lm_head on llama3.

### 4. How to test?
- [ ] N/A
- [ ] Unit test: Please manually trigger the PR Validation [here](https://github.com/intel-analytics/ipex-llm-workflow/actions/workflows/llm-PR-validation.yml) by inputting the PR number (e.g., `1234`). And paste your action link here once it has been successfully finished.
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

**Comment by jason-dai** [comment]
Do we need to set transpose-v-cache to false for new drivers?

**Comment by cyita** [comment]
> Do we need to set transpose-v-cache to false for new drivers?

llama3 transpose-v-cache=True
minicpm-2b transpose-v-cache=False

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md [hunk 1]
```diff
@@ -23,6 +23,12 @@ Go to https://www.intel.com/content/www/us/en/download/794734/intel-npu-driver-w
 Then go to **Device Manager**, find **Neural Processors** -> **Intel(R) AI Boost**.
 Right click and select **Update Driver**. And then manually select the folder unzipped from the driver.
 
+### Recommended NPU Driver Version for LNL users
+#### 32.0.100.2625
+Supported models: Llama2-7B, Qwen2-1.5B, MiniCPM-1B, Baichuan2-7B
+#### 32.0.101.2715
+Supported models: Llama3-8B, MiniCPM-2B
+
```
Comment (jason-dai) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:31]: Move this to section 4

**Review by sgwhat — APPROVED** [review]
LGTM

**Pull request merged**